test/multiple-user-principal-types.test.js

```diff
@@ -373,19 +383,36 @@ describe('Multiple users with custom principalType', function() {
 
         return Album.create({name: 'album', userId: userFromOneModel.id})
           .then(function(album) {
-            return Role.isInRole(
-              Role.OWNER,
-              {
-                principalType: OneUser.modelName,
-                principalId: userFromOneModel.id,
-                model: Album,
-                id: album.id,
-              });
+            baseContext = {
+              principalId: userFromOneModel.id,
+              model: Album,
+              id: album.id,
+            };
+            var validContext = buildContextWith({principalType: OneUser.modelName});
+            var invalidContext = buildContextWith({principalType: AnotherUser.modelName});
+
+            return isInRole([
+              {role: Role.OWNER, context: validContext},
+              {role: Role.OWNER, context: invalidContext},
+            ]);
           })
-          .then(function(isInRole) {
-            expect(isInRole).to.be.true();
+          .then(function(data) {
+            expect(data).to.eql([true, false]);
```

Comment:
This assertion produces an error message that's difficult to understand without reading the source code of the test. Can you please split the test into two, where each tests asserts a single scenario (valid context, invalid context)?